### PR TITLE
Add run-script support (solves #54)

### PR DESCRIPTION
**Pull request by kevroadrunner**
This PR adds the `oao run-script` command, familiar to `oao all`.

### CODE REVIEW

**Comment by kevroadrunner** [comment]
(solves #54)

File: README.md [hunk 1]
```diff
@@ -210,6 +210,17 @@ $ oao all "yarn run compileWatch" --parallel
 
 **Note: some terminals may have problems with parallel logs (based on [terminal-kit](https://github.com/cronvel/terminal-kit)). If you experience issues, use the `--no-parallel-logs` flag. If you're using the default terminal or Hyper on OS X or Windows, you should be fine.**
 
+### `oao run-script <script>`
+
+Executes the specified (package) script on all sub-packages. Examples:
```
Comment (guigrpa) [inline README.md]: Maybe you could mention that sub-packages that don't have the specified script are skipped.

File: src/__tests__/runScript.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* eslint-env jest */
+/* eslint-disable global-require */
+
+import runScript from '../runScript';
+
+jest.mock('../utils/shell');
+
+describe('ALL command', () => {
```
Comment (guigrpa) [inline src/__tests__/runScript.test.js]: Typo: RUN-SCRIPT command (please fix, and don't forget to update the corresponding snapshot)

**Comment by guigrpa** [comment]
That was fast, thanks a lot!

**Pull request merged**